src/MultiTypeSymbolTable.chpl

```diff
@@ -390,6 +390,12 @@ module MultiTypeSymbolTable
                 var g:SegStringSymEntry = toSegStringSymEntry(entry);
                 return "%s %s %t %t %t %t".format(name, dtype2str(g.dtype), g.size, g.ndim, g.shape, g.itemsize);
             }
+            else if entry.isAssignableTo(SymbolEntryType.SegArraySymEntry) {
+                var g:CompositeSymEntry = toCompositeSymEntry(entry);
```

Comment:
Currently this is not necessary because we can access all the data needed from the CompositeSymEntry. It may be required later on, but currently it is not.

And again, `toSegArraySymEntry` cannot be created because the return type cannot be determined due the `SegArraySymEntry` being a generic type.